### ISSUE TITLE
Unwanted space above the banner

### ISSUE THREAD

**Issue by thomasvs** [noatpad/obsidian-banners#38]
Hi,

neat plug-in, thanks!

I'm trying to figure out why there's space above my banners, and an expander handle that I'd like to get rid of.

I think the expander handle goes away by choosing the embed style, is that right?

I can't get rid of the extra size, and I narrowed it down to one css rule, but I don't know why it's happening or how I can fix it.

Attaching screenshots of the one css rule enabled/disabled if that helps.
![Screenshot from 2022-02-17 17-09-14](https://user-images.githubusercontent.com/248196/154579931-fa8b9e9d-7db4-4248-8a00-6558b77c15c0.png)
![Screenshot from 2022-02-17 17-09-21](https://user-images.githubusercontent.com/248196/154579933-331f942d-9b5d-4793-9d91-a412cc00bd91.png)

**Comment by noatpad** [comment]
Huh, is the theme perhaps?
That one CSS rule is actually intended to be there, but the child with the class name `obsidian-banner` should have a `position; absolute;` rule among others that sets the banner in place. Just to make sure, which theme/plugins are you using?

**Comment by lepperm** [comment]
@thomasvs It looks like this might be coming from the [Copy button for code blocks](https://github.com/jdbrice/obsidian-code-block-copy) plugin. I noticed this in my vault as well, and disabling the copy button fixed my issues.

**Comment by joethei** [comment]
Just found this out to, that plugin adds it's a bit of css.
This issue https://github.com/jdbrice/obsidian-code-block-copy/issues/17 has a workaround

**Comment by noatpad** [comment]
Yeah this looks like that's one of those plugin compatibility issues. The workaround mentioned should work fine for now since I'm considering doing some changes to fix those conflicts
